Training.Core/Models/Album.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Training.Core.Contracts;
+
+namespace Training.Core.Models
+{
+    public class Album : IRemovable
+    {
+
+        public Guid Id { get; set; }
+        public string ISBN { get; set; }
```

Comment:
ISBN has sense as a unique global identifier for a Book, we have to found something different like a code for Album